ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -336,20 +359,24 @@ runDataDiffusion tracers
          )
 
     runServer :: SocketSnocket -> NetworkMutableState SockAddr -> Either Socket.Socket SockAddr -> IO ()
-    runServer sn networkState address =
+    runServer sn networkState address = do
+      traceWith dtDiffusionInitializationTracer DiffusionInitiatilizationRunServer
```

Comment:
I would pus this message down, just before calling `withServer`. It would be nice to include the address in the message.  You can get it from `Either Socket SockAddr` using `getSocketName :: Socket -> IO SockAddr` (to get the address of the systemd socket).